Gemfile

```diff
@@ -15,7 +15,6 @@ gem "mongoid"
 gem "plek"
 gem "responders"
 gem "sass-rails"
-gem "sdoc", group: :doc
```

Comment:
I think this change should probably be split out into a separate commit (or at least included in the commit message).

Reply:
Oops, that must have got squashed accidentally in a rebase.

I've moved it into a separate commit 👍.